tools/pipelines/templates/include-test-real-service.yml

```diff
@@ -334,6 +334,17 @@ jobs:
           mergeTestResults: false
         condition: succeededOrFailed()
 
+      # Publish tinylicious log for troubleshooting
+      - ${{ if contains(convertToJson(parameters.testCommand), 'tinylicious') }}:
+        - task: PublishPipelineArtifact@1
+          displayName: Publish Artifact - Tinylicious Log
+          inputs:
+            targetPath: '${{ parameters.buildDirectory }}/packages/test/test-end-to-end-tests/tinylicious.log'
```

Comment:
I think this needs to be something like `${{ parameters.testWorkspace }}/node_modules/${{ parameters.testPackage }}/tinylicious.log`. The template is used by both e2e and stress tests and they end up having different paths (reflected in `parameters.testPackage`); I imagine it'll also apply to azure e2e tests.